ros/src/computing/perception/detection/packages/road_wizard/launch/feat_proj.launch

```diff
@@ -1,10 +1,14 @@
 <launch>
-  <arg name="camera_id" default="/"/>
+  <arg name="camera_id" default="/camera0"/>
   <arg name="camera_info_src" default="/camera_info"/>
   <arg name="use_path_info" default="false"/> <!-- USE VectorMap Server to publish only TrafficSignals on current lane-->
+  <arg name="use_filtered_signal_map" default="false"/>
+  <arg name="filtered_signal_map_name" default="$(find demo_scripts)/config/filteredsignaldata.csv"/>
```

Comment:
please add a comment here why we introduced this:
Because computing lane to signal light connection is too slow in this function: https://github.com/CPFL/Autoware/blob/feature/pacmod_apex/ros/src/data/packages/vector_map_server/nodes/vector_map_server/vector_map_server.cpp#L357-L363 we go around the problem by providing a list of lights that are on our waypoint. Note: This is a semi-hack that will only work for our demo. If you record different waypoints on different lanes, you need to recreate filteredsignaldata.csv.

To make the proper fix we would need to use a different data structure instead of https://github.com/CPFL/Autoware/blob/feature/pacmod_apex/ros/src/data/packages/vector_map/include/vector_map/vector_map.h#L362 for faster querying.

Reply:
done